src/EnergyPlus/CostEstimateManager.hh

```diff
@@ -200,14 +191,7 @@ struct CostEstimateManagerData : BaseGlobalStruct
 
     void clear_state() override
     {
-        this->GetCostInput = true;
-        this->NumLineItems = 0;
-        this->DoCostEstimate = false;
-        this->numMonetaryUnit = 0;
-        this->selectedMonetaryUnit = 0;
-        this->CurntBldg = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0};
```

Comment:
You didn't do anything wrong here, but I dislike that the default values on the struct are all 0.0, but in both cases where we instantiate them, we pass a 1.0 as one of the arguments.  I think we should clean that up and just default assign it to 1.0 in the struct and then we don't have to have this initializer list anymore.